charts/redpanda/values.yaml

```diff
@@ -344,6 +344,10 @@ resources:
     #
     # This setting is equivalent to `--smp`, `resources.requests.cpu`, and `resources.limits.cpu`.
     # For production, use `4` or greater.
+    #
+    # To maximise efficiency and assign Redpanda Pods to CPU cores you can use Static CPU manager Policy by specifying
+    # integer CPU in resources requests and limits that is even.
+    # https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/#static-policy
```

Comment:
```suggestion
    # To maximize efficiency, use the `static` CPU manager policy by specifying an even integer for 
    # CPU resource requests and limits. This policy gives the Pods running Redpanda brokers 
    # access to exclusive CPUs on the node. See https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/#static-policy.
```

Reply:
do they get access to the specified number of CPUs on the node?

Reply:
Yes, only if Kubelet is correctly configured with static CPU policy. 

https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/#configuration